docs/diagrams/resident-sequence-flow.iuml

```diff
@@ -0,0 +1,21 @@
+@startuml
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/v2.2.0/C4_Deployment.puml
+!include https://raw.githubusercontent.com/LBHackney-IT/cv-19-res-support-v3/development/docs/diagrams/c4_shared.puml
+
+title Resident sequence flow diagram
+
+Resident -> Resident: Receive email
+Resident -> Next.JS: Click link in email
+Next.JS -> EvidenceAPI: GET /evidence_requests/:id
+Next.JS <- EvidenceAPI: 200 Evidence request found
+Resident <- Next.JS: Show instructions
+Resident -> Next.JS: Click continue
```

Comment:
Do we want to mention the Resident needs to _Attach and upload file(s)_ here?

Reply:
Sure, I'll change it